lib/dependency-manager-adapters/npm.js

```diff
@@ -16,6 +16,8 @@ module.exports = CoreObject.extend({
   },
   useYarnCommand: false,
   yarnLock: 'yarn.lock',
+  yarnLockBackupFileName: 'yarn.lock.ember-try',
+  useYarnLockFile: false,
```

Comment:
I will leave it up to @rwjblue, but `ember-cli`, and consequently `ember-try` as well, supports both `npm` and `yarn`. If we are introducing `useLockfile` option, which should apply to both NPM and Yarn, we should probably handle it for both instead of making the option only valid in one case.

Reply:
@dnachev Yea, I was contemplating whether to support both cases. I ultimately decided to support the yarn case only and named the flag useYarnLockFile instead of useLockFile. 

If we want to support using shrinkwrap, then we can add support for this in the future if there is a need? I was thinking this would be a separate flag called useNPMShrinkwrap or something along this line.